coral/Makefile

```diff
@@ -10,7 +10,7 @@ default: $(EMBEDDED_INDEX_HTML)
 	@echo "TODO: ADD USAGE INSTRUCTIONS"
 
 $(CORAL_DIST_PATH): index.html $(wildcard src/**/*)
-	pnpm build --assetsDir assets/coral --mode springboot
+	../core/nodebuild/node/node_modules/.bin/pnpm build --assetsDir assets/coral --mode springboot
```

Comment:
@SmuliS do we still need this path for pnpm ?

Reply:
You can remove all changes from this file. We created the new `Makefile` to `core` so we would not need to have these ugly executable references.